vendor/nanogui/common.cpp

```diff
@@ -63,9 +63,10 @@ void init() {
 }
 
 static bool mainloop_active = false;
-bool pollEvents = false;
+int throttle = 0;
 
 void mainloop(int refresh) {
+    throttle = refresh;
```

Comment:
Just wrap this with `#ifndef WIN32`

Reply:
that's not quite gonna work because it can still be overridden by the app... I know where it needs to go...

Reply:
Added to `CAvaraGame::GameStart` which is where the refresh rate is sped up just before start/resume of a game.